autopts/ptsprojects/mynewt/gap_wid.py

```diff
@@ -76,12 +72,33 @@ def hdl_wid_204(_: WIDParams):
     return btp.check_discov_results(addr_type=0x02)
 
 
+def hdl_wid_227(params: WIDParams):
+    bd_addr = btp.pts_addr_get()
+    bd_addr_type = btp.pts_addr_type_get()
+    stack = get_stack()
+
+    handle = btp.parse_handle_description(params.description)
+    if not handle:
+        return False
+
+    btp.gatt_cl_write(bd_addr_type, bd_addr, handle, '02', 1)
```

Comment:
maybe handle this in generic wid?

if (gatt_cl_supported)
   gatt_cl...
else
  gattc....

Reply:
hmm I think I missed this, general wid works ok here. I'll remove this custom one